TypeScript throws compile error TS2714 in dist/htm.d.ts

Hi, thanks to all of great work for HTM! I'm loving to integrate htm with our library.

I tried to bump htm to v3.0.2 but TypeScript 3.7.5 throws a compile error `TS2714: The expression of an export assignment must be an identifier or qualified name in an ambient context.` in `dist/htm.d.ts`.

```
yarn check:ts
yarn run v1.21.1
$ tsc --noEmit
node_modules/htm/dist/htm.d.ts:1:16 - error TS2714: The expression of an export assignment must be an identifier or qualified name in an ambient context.

  1 export default {
                   ~
  2   bind<HResult>(
    ~~~~~~~~~~~~~~~~
... 
  4   ): (strings: TemplateStringsArray, ...values: any[]) => HResult | HResult[];
    ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
  5 };
    ~

node_modules/htm/dist/htm.d.ts:4:78 - error TS1005: '{' expected.

4   ): (strings: TemplateStringsArray, ...values: any[]) => HResult | HResult[];
                                                                               ~


Found 2 errors.

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

ref: https://github.com/speee/jsx-slack/pull/107, https://circleci.com/gh/speee/jsx-slack/1294

--- a/src/index.d.ts
+++ b/src/index.d.ts
@@ -1,5 +1,6 @@
-export default {
+declare const htm: {
   bind<HResult>(
     h: (type: any, props: Record<string, any>, ...children: any[]) => HResult
   ): (strings: TemplateStringsArray, ...values: any[]) => HResult | HResult[];
 };
+export default htm;

